include/zephyr/bluetooth/conn.h

```diff
@@ -252,6 +252,93 @@ struct bt_conn_le_subrate_changed {
 	uint16_t supervision_timeout;
 };
 
+/** Most Connection Interval Groups possible
+ *
+ * The practical maximum is 41 groups based on HCI event size constraints:
+ *     (HCI event max size - cmd_complete overhead - response fields) / sizeof(group)
+ *     (255 - 4 - 3) / 6 = 41 groups max
+ */
+#define BT_CONN_LE_MAX_CONN_INTERVAL_GROUPS 41
+
+/** Minimum supported connection interval group
+ *
+ * Each group represents an arithmetic sequence of supported connection intervals:
+ *   min, min+stride, min+2*stride, ..., min+n*stride (where min+n*stride <= max)
+ *
+ * Example: min=10 (1250 us), max=60 (7500 us), stride=5 (625 us)
+ *   -> Supported: 1250 us, 1875 us, 2500 us, ..., 6875 us, 7500 us
+ *
+ * All values in units of 125 microseconds.
+ */
+struct bt_conn_le_min_conn_interval_group {
+	/** Lower bound of group interval range */
+	uint16_t min;
+	/** Upper bound of group interval range */
+	uint16_t max;
+	/** Increment between consecutive supported intervals */
+	uint16_t stride;
+};
+
+/** Minimum supported connection interval information */
+struct bt_conn_le_min_conn_interval_info {
+	/** Minimum supported connection interval in units of 125 microseconds. */
```

Comment:
We should try to be consistent about units. Is there a reason this field is in `125us` which does not apply to `bt_conn_le_info.interval_us`?

Reply:
125 us is the base unit of SCI. HCI commands send and receive SCI timings in units of 125 us. I chose to use interval_us instead of in units of 125 because it is impartial to the base used and the timing represented is intuitive to understand.